lib/puppet/pops/parser/locator.rb

```diff
@@ -261,13 +261,13 @@ def line_for_offset(offset)
       end
       if line_nbr = ary_bsearch_i(line_index, offset)
         # cache
-        prev_offset = offset
-        prev_line = line_nbr
+	      self.prev_offset = offset
+        self.prev_line = line_nbr
```

Comment:
@hlindberg @thallgren It appears the Locator never caches anything, because it's creating local variables named `prev_offset` and `prev_line` that shadow the accessor methods of the same name. Can you point me to where tests for this functionality live?

Reply:
Once upon a time it was at least verified to have performance impact, but doubt there are actual tests to prevent regression.

Reply:
@joshcooper tab on line 264

Reply:
Filed as PUP-8247. I'll remove my changes, and add a disable pragma for the check.